src/libs/antares/array/matrix.hxx

```diff
@@ -1571,6 +1578,25 @@ inline Matrix<T, ReadWriteT>& Matrix<T, ReadWriteT>::operator=(const Matrix<T, R
     return *this;
 }
 
+template<class T, class ReadWriteT>
+inline Matrix<T, ReadWriteT>& Matrix<T, ReadWriteT>::operator=(Matrix<T, ReadWriteT>&& rhs) noexcept
+{
+    width = rhs.width;
+    height = rhs.height;
+    jit = rhs.jit;
+    if (0 == width || 0 == height)
+    {
+        entry = nullptr;
+        width = 0;
+        height = 0;
+    }
+    else
+    {
+        entry = rhs.entry;
```

Comment:
Question : I don't master move semantics, but since **rhs** is "moved", shouldn't we  leave **rhs.entry** to **nullptr** at the end of the copy operator ?

Reply:
ok